src/App.css

```diff
@@ -0,0 +1,9 @@
+@import "tailwindcss";
+@plugin "daisyui";
+
+#root {
+  margin: 0 auto;
+  padding: 2rem;
+  text-align: left;
+  background-color: rgb(210, 210, 210);
```

Comment:
Right now, when the are no contacts, the background color only spans a part of the page. You can make it span the entire window by adding a minimum height of 100% of the viewport height: `min-height: 100vh;`

<img width="1440" alt="image" src="https://github.com/user-attachments/assets/f7ddb415-529e-4808-acdf-544a5a8f35f7" />